M2/Macaulay2/m2/packages.m2

```diff
@@ -553,6 +555,7 @@ debug GlobalDictionary := dict -> (
     checkShadow())
 
 packageFiles = pkg -> (
+    if pkg#"pkgname" == "User" then return {};
```

Comment:
Is this the only package that is problematic?
How about a package defined directly in the interpreter?

Reply:
Good point!

```m2
i2 : newPackage "Foo"

o2 = Foo

o2 : Package

i3 : endPackage "Foo"

o3 = Foo

o3 : Package

i4 : packageFiles oo

o4 = {/home/profzoom/src/macaulay2/M2/M2/Macaulay2/m2/stdio}

o4 : List
```

I'll play around with this some more

Reply:
This should be addressed now